apps/dashboard/src/main/resources/SampleApiData.json

```diff
@@ -2648,6 +2648,162 @@
       },
       "statusCode": 200
     }
+  },
+  {
+    "id": "INVALID_SSN",
+    "sampleData": {
+      "method":"GET",
+      "requestPayload": "",
+      "responsePayload": "{\"status\" : \"SSN validation complete\"}",
```

Comment:
Just check if we can add some relevant request payload, api would look a little more legit.